ui/src/pages/CreateTrip.jsx

```diff
@@ -80,14 +91,26 @@ class CreateTripPage extends React.Component {
   }
 }
 
+CreateTripPage.propTypes = {
+  addEvent: PropTypes.bool,
```

Comment:
I don't think `addEvent` is ever used as a prop here (or passed in as a prop either). We can probably get rid of the `propTypes` and `defaultProps`